backend/permissions/permission.py

```diff
@@ -18,11 +18,29 @@ class IsOwnerOrSharedUser(permissions.BasePermission):
     """Custom permission to only allow owners and shared users of an object."""
 
     def has_object_permission(self, request: Request, view: APIView, obj: Any) -> bool:
+
+        return (
+            True
+            if (
+                obj.created_by == request.user
+                or obj.shared_users.filter(pk=request.user.pk).exists()
+            )
+            else False
+        )
+
+
+class IsOwnerOrSharedUserOrOrg(permissions.BasePermission):
```

Comment:
@johnyrahul Can we modify the naming here ? This class basically allows permission to all part of the org. right, so name can be made better to get more clarity ?